### PR TITLE
🦄NEW: Add Shades-of-Purple theme

### PR DESCRIPTION
**Pull request by ahmadawais**
Add the `json` object for Shades-of-Purple theme 💜

I was [asked on Twitter](https://twitter.com/markdrew53/status/986101034031419392) to add the theme here. So, fingers crossed. 🤞

---

### Details:

image

Name: `shades-of-purple-hyper`
Description: `Professional Hyper™ Terminal theme with hand-picked & bold shades of purple 💜 to go along with your command line usage. Love purple, coz. purple can be genius.`
  
COLOR | HEX | 
--- | --- | 
Background | #2D2B55 | 
Lite Purple | #6943FF | 
Contrast Yello | #FAD000 | 
Contrast Pink | #FF2C70 | 
Shade of Purple | #B362FF | 

Peace! ✌️

### CODE REVIEW

**Comment by MAPESO** [comment]
@ahmadawais Hi, I see that you added a video, what are themes are static images, I know that this does not specify in this [link](https://github.com/zeit/hyper-site/wiki/Submitting-a-new-plugin-or-theme-to-Hyper-Store) but all themes that is in hyper store is following a design guideline like this.

hyper-solarized-light

**Comment by ahmadawais** [comment]
Mark you want me to change the screenshot?! It's written in the docs that
it can be a gif. Also restricting the screenshots to all look the same
would be limiting. Users mostly pick the themes based on how well and
different they're presented.

Thoughts?! 🙈👀

File: plugins.json [hunk 1]
```diff
@@ -179,5 +179,19 @@
...
     ],
     "dateAdded": "1523900385213"
-  }
+  },
+  {
+    "name": "shades-of-purple-hyper",
+    "description": "Professional Hyper™ Terminal theme with hand-picked & bold shades of purple 💜 to go along with your command line usage. Love purple, coz. purple can be genius.",
+    "type": "theme",
+    "preview": "https://on.ahmda.ws/qvUU/c",
+    "colors": [,
```
Comment (timothyis) [inline plugins.json]: The comma at the end here isn't valid json, please remove it 🙏

File: plugins.json [hunk 2]
```diff
@@ -179,5 +179,19 @@
...
+      "#2D2B55",
+      "#6943ff",
+      "#FAD000",
+      "#ff2c70",
+      "#b362ff"
+    ],
+    "dateAdded": "1523965034087"
+}
```
Comment (timothyis) [inline plugins.json]: Can you indent this by 2 spaces?

File: plugins.json (hunk 1 above)
Comment (ahmadawais) [reply plugins.json]: Sure.

**Comment by ahmadawais** [comment]
@timothyis I just fixed the silly mistakes. Apologies for that.

Review now :)

Also, I am not using any other plugins but just a simple script to show-off theme colors.

**Comment by timothyis** [comment]
Thank you!

**Comment by ahmadawais** [comment]
@timothyis Awesome! 💯
When will it go live on the site? Just wondering. Also, how does a theme gets featured? If mine did, I'd love to promote that to my [Twitter](https://twitter.com/mrahmadawais/) and [newsletter](https://WPTakeaway.club/) audience of 100,000+ developers.

Looking forward!
Peace! ✌️

**Comment by timothyis** [comment]
@ahmadawais It'll go live today, I'm planning on making the release automatic.

For featured plugins/themes, we'll probably review them bi-weekly or monthly to decide which should be featured. We'll let you know though.

**Comment by ahmadawais** [comment]
Awesome 💯
Looking forward to it.

Peace! ✌️

**Comment by timothyis** [comment]
Hi @ahmadawais. I've had to remove your theme from being listed for now due to an error with the image you submitted being flagged as phishing. Due to the preloading nature of the Hyper themes list, it caused the site to error when loading your image. If you'd like to submit a new link, I'd be happy to add it back.

**Comment by ahmadawais** [comment]
Sure, I will do that. I use this service call https://www.getcloudapp.com/ and it has been used by some bad actors to do some bad shit which caused Google to penalize all of the links — such a d* move on Google's part to destroy millions of users and their online properties.

**Comment by ahmadawais** [comment]
@timothyis all that needs to be done is to replace the image link with this 

```
https://raw.githubusercontent.com/ahmadawais/Shades-of-Purple-Hyper/master/images/Shades-of-Purple-Hyper.gif
```

**Pull request merged**